charts/stable/multus/templates/uninstallJob.yaml

```diff
@@ -22,11 +22,11 @@ spec:
       - name: remove-config
         image: "alpine"
         command: ["/bin/sh","-c", "rm -rf  /host/etc/cni/net.d/*multus*"]
-        {{- with (include "common.controller.volumeMounts" . | trim) }}
         volumeMounts:
-          {{- . | nindent 10 }}
-        {{- end }}
-      {{- with (include "common.controller.volumes" . | trim) }}
+        - name: cni
+          mountPath: /host/etc/cni/net.d
       volumes:
-        {{- . | nindent 8 }}
-      {{- end }}
+      - name: cni
+        hostPath:
+          path: /var/lib/rancher/k3s/agent/etc/cni/net.d
```

Comment:
Could it be that this needs to refer to `{{ cni.paths.config }}`?

Reply:
You are right - changed.